server/utils/errorHandler.js

```diff
@@ -1,5 +1,7 @@
 export const errorHandler = ({ error, functionName, message, req, res }) => {
   try {
+    if (process.env.NODE_ENV === "development") console.log("Error:", error);
```

Comment:
this should be console.error

there are many places in the code using console.log for logging errors. It should be console.error

![image](https://github.com/metaversecloud-com/sdk-quest/assets/9873724/3af59567-ecb2-4ae4-a35c-707f77f7044f)